package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "egghead-ui",
-  "version": "1.0.7",
+  "version": "1.1.0",
```

Comment:
@trevordmiller Weird, maybe it jumped like that because I didn't pull while we were working??